mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -294,6 +296,18 @@ final class OneVsRest @Since("1.4.0") (
   @Since("1.5.0")
   def setPredictionCol(value: String): this.type = set(predictionCol, value)
 
+  /** @group expertGetParam */
+  override def getParallelism: Int = $(parallelism)
+
+  /**
+   * @group expertSetParam
+   * The implementation of parallel one vs. rest runs the classification for
```

Comment:
This doc could be a bit better - something like "Set the number of threads used to fit the model for each class in parallel. Default is 1 (fit all models serially)." or similar